docs/api/search-query-service-resource.md

```diff
@@ -105,6 +110,7 @@ tags           | string or array of strings | no       |
 title          | string                     | no       | 
 totalDownloads | integer                    | no       | This value can be inferred by the sum of downloads in the `versions` array
 verified       | boolean                    | no       | A JSON boolean indicating whether the package is [verified](../nuget-org/id-prefix-reservation.md)
+packageTypes   | array of objects           | yes      | The package types defined by the package author
```

Comment:
Most NuGet V3 search implementations (like Azure DevOps, BaGet, Sleet) do not provide the `packageTypes` property. I would either make this not required, or, add in the notes that the property was added by search v3.5.0